srcpkgs/gimx/patches/musl.patch

```diff
@@ -0,0 +1,70 @@
+Upstream: Sent but no response received
```

Comment:
Do you have a PR?

Reply:
No I emailed the patch. I could try a PR.